packages/legend-graph/src/graph-manager/protocol/pure/v1/engine/V1_Engine.ts

```diff
@@ -752,17 +766,25 @@ export class V1_Engine {
   async runQueryAndReturnString(
```

Comment:
Maybe since this is no longer returning a string, we should rename it?
```suggestion
  async runQueryAndReturnMap(
```